### PR TITLE
Revert "Adding RBC study"

### PR DESCRIPTION
**Pull request by AnneCarpenter**
Reverts greenelab/deep-review#815

Sorry to create more work, but we plan some major revisions to this paper and I'd rather not promote it in its current state. Let's not cite it for now, after all.

### CODE REVIEW

File: content/04.study.md [hunk 1]
```diff
@@ -347,9 +347,9 @@ A field poised for dramatic revolution by deep learning is bioimage analysis.
 Thus far, the primary use of deep learning for biological images has been for segmentation---that is, for the identification of biologically relevant structures in images such as nuclei, infected cells, or vasculature---in fluorescence or even brightfield channels [@doi:10.1371/journal.pcbi.1005177].
 Once so-called regions of interest have been identified, it is often straightforward to measure biological properties of interest, such as fluorescence intensities, textures, and sizes.
 Given the dramatic successes of deep learning in biological imaging, we simply refer to articles that review recent advancements [@doi:10.3109/10409238.2015.1135868; @doi:10.1371/journal.pcbi.1005177; @doi:10.1007/978-3-319-24574-4_28].
-For deep learning to become commonplace for biological image segmentation, however, user-friendly tools must be developed.
+For deep learning to become commonplace for biological image segmentation, we need user-friendly tools.
```
Comment (cgreene) [inline content/04.study.md]: Do we want to preserve this change? I trust your call on reverting the reference below.

Comment (AnneCarpenter) [reply content/04.study.md]: I don't mind either way, I suppose keeping the edit would be nice (if you can make that change yourself? Not sure how I'd do it, sadly!

**Comment by cgreene** [comment]
Let me know if that looks good. If so, I'll squash/merge.

**Comment by AnneCarpenter** [comment]
Perfect! Thanks.

**Pull request merged**